src/js/edu-benefits/education-wizard.js

```diff
@@ -89,4 +89,14 @@ function reInitWidget() {
   resetForm();
 }
 
-document.addEventListener('DOMContentLoaded', reInitWidget);
+function loadImports() {
+  const link = document.querySelector('link[rel=import]');
```

Comment:
Can we add an id to this `<link>` element? Again, hedging for the future when we may have multiple import links.